src/Terrain/Maps/IMapBuilder.java

```diff
@@ -25,5 +26,5 @@
   
   protected static final TerrainType LB = TerrainType.HEADQUARTERS; // lab -> HQ
   protected static final TerrainType SR = TerrainType.BUNKER; // full silo, but empty because lol
-  protected static final TerrainType TW = TerrainType.BUNKER; // tower
+   // tower
```

Comment:
Rather than deleting the line, you should set the TW abbreviation to TerrainType.TOWER